src/sas/qtgui/Calculators/DataOperationUtilityPanel.py

```diff
@@ -126,7 +130,7 @@ def onClose(self):
 
 
     def onCompute(self):
-        """ perform calculation """
+        """ perform calculation - don't send to data explorer"""
```

Comment:
You decoupled `compute` and `save` but gave no visual cue to the user that the calculation succeeded. Previously, getting a new item in the DE was a GUI hint.
Consider adding a message to the status bar.

Reply:
This is a great idea! Warnings and info statements have been added to logging that let the user know whether interpolation was performed on Data2 and that the computation was successful.